modules/flash-script.nix

```diff
@@ -198,9 +198,12 @@ in
         eksFile = mkOption {
           type = types.nullOr types.path;
           default = null;
+          description = "
```

Comment:
This should be in a different PR.
Also, it would need to be triple quotes, instead of double quotes.